pytorch_lightning/trainer/training_loop.py

```diff
@@ -515,8 +516,22 @@ def run_training_batch(self, batch, batch_idx):
         for split_idx, split_batch in enumerate(splits):
             self.split_idx = split_idx
 
+            def get_optimizers_iterable():
+                if not self.optimizer_frequencies:
+                    return enumerate(self.optimizers)
```

Comment:
```suggestion
                    return list(enumerate(self.optimizers))
```
to be list not iterator...

Reply:
before these changes, we would use `for... in enumerate (self.optimizers):`
In this case, where the list is so short, I don't see the difference.
Will it affect runtime?